node_asg.tf

```diff
@@ -3,11 +3,11 @@
 // Setup for demo/testing purposes.
 resource "aws_autoscaling_group" "node" {
   name                      = "${var.cluster_name}-node"
-  max_size                  = 1000
-  min_size                  = 1
+  max_size                  = 0
+  min_size                  = 0
```

Comment:
I would greatly prefer is we could completely disable the `node` ASG and all associated resources with a parameter to the module.